playbooks/osp/12/undercloud/undercloud_post_install_osp.sh.j2

```diff
@@ -26,25 +26,12 @@ openstack overcloud node import /home/stack/instackenv.json
 # set dns servers
 openstack subnet set --dns-nameserver 8.8.8.8 --dns-nameserver 8.8.4.4 ctlplane-subnet
 
-# Tagging nodes roles and names, we require to know the servers IDs first
-openstack baremetal node list | grep controller1 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:control,boot_option:local' '{}'
-openstack baremetal node list | grep controller2 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:control,boot_option:local' '{}'
-openstack baremetal node list | grep controller3 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:control,boot_option:local' '{}'
-openstack baremetal node list | grep compute1 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:compute,boot_option:local' '{}'
-openstack baremetal node list | grep compute2 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:compute,boot_option:local' '{}'
-openstack baremetal node list | grep ceph1 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:ceph-storage,boot_option:local' '{}'
-openstack baremetal node list | grep ceph2 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:ceph-storage,boot_option:local' '{}'
-openstack baremetal node list | grep ceph3 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:ceph-storage,boot_option:local' '{}'
-
-#openstack baremetal node list | grep swift1 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:swift-storage,boot_option:local' '{}'
-#openstack baremetal node list | grep swift2 | awk '{print $2}' | xargs -I '{}' openstack baremetal node set --property capabilities='profile:swift-storage,boot_option:local' '{}'
-
 # Configuring the Overcloud to Use the Undercloud as a Local Registry
 LATEST_IMAGE_TAG=`sudo openstack overcloud container image tag discover \
```

Comment:
if you remove the use of sudo below, is it not also (technically) possible to remove it here?

Reply:
argh, i'll add the comment here (because github won't let me make it below where appropriate because it's apparently too far out of context of this PR's change)...

can the `sudo` used with `openstack overcloud container image upload` be removed the same as here?

Reply:
Yeah, I'm not sure if upload will work without sudo, the docs mention sudo for the image upload but not for the prepare (https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html/director_installation_and_usage/configuring-a-container-image-source#container-images-additional-services).  I ran into an issue with the prepare because it was creating the images file with root perms when using sudo so I dropped it.

Yeah, I haven't tested if those instackenv changes work for OSP 12 yet, should be able to do it today though and verify.  It's much cleaner so I'm hoping it'll work. :)

Reply:
Looks like this works and passes in 12, so merging in.